buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchJavaPlugin.java

```diff
@@ -145,6 +160,7 @@ public static void configureConfigurations(Project project) {
         disableTransitiveDeps.accept(JavaPlugin.TEST_COMPILE_CONFIGURATION_NAME);
```

Comment:
We can probably remove this now since we fail if you add any dependencies to this configuration.